README.md

```diff
@@ -2,15 +2,55 @@
+
+You can find the rooms by adding the 
+
+## Status of the project
+
+At the moment Haveno is simply a Proof Of Concept. It's already possible to initiate a trade, but the project still needs a lot of work before being available for public use.
+
+These are the main priorities for the near future:
+
+- We need a logo. Right now it's a random stock image :)
+- The User Interface is basically still Bisq. The entire interface needs to be completely reworked and adapted for Monero as base currency
+- Create documentation for developers
+- Cleanup the repository from Bisq-specific content
+- Organzie what Bisq calls "Arbitrators"
+
+See the issue trackers for more detailed info. 
+
+## Why a new platform?
+
+We believe Bisq is not enough for Monero users, which want a private way to exchange Monero for other (crypto)currencies. Haveno is built on Monero, which means all transactions between users are obfuscated by default. Bisq's system is based on Bitcoin and inherits all its design vulnerabilities, for example:
+
+- All Bisq's in-platform transactions are based on Bitcoin, so fully traceable.
+- Bisq transactions are unique and easily visible on the blockchain. This means it's trivial to check which Bitcoin transactions are the result of a trade on Bisq.
+
+Even if XMR transactions compose the vast majority of Bisq's activity, the team never displayed much interest in improving the Monero integration. The important privacy issues mentioned above will be solved by simply having Monero as a base currency instead of Bitcoin.
+
+We acknowledge and thank Bisq for their efforts, but we think the Monero community needs a native, private way to exchange XMR for other currencies without passing through Bitcoin first and Haveno is here to fill that gap! We commit to contribute back to Bisq when possible.
+
+## FAQ
+
+See the Wiki on this repository (TODO)
 
 ## Running a local test network
 
 1. Download [Monero CLI](https://www.getmonero.org/downloads/) for your system
-2. Sync stagenet: `./monerod --stagenet`, or alternatively, start a local stagenet network:
+2. Sync stagenet: `./monerod --stagenet --rpc-login superuser:abctesting123 --confirm-external-bind`, or alternatively, start a local stagenet network:
     * `./monerod --stagenet --no-igd --hide-my-port --data-dir node1 --p2p-bind-ip 127.0.0.1 --p2p-bind-port 48080 --rpc-bind-port 48081 --zmq-rpc-bind-port 48082 --add-exclusive-node 127.0.0.1:38080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080`<br>
     * `./monerod --stagenet --no-igd --hide-my-port --data-dir node2 --p2p-bind-ip 127.0.0.1 --rpc-bind-ip 0.0.0.0 --confirm-external-bind --add-exclusive-node 127.0.0.1:48080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080`
-3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
+3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download) (needed if you want to test a XMR <-> BTC exchange)
```

Comment:
Pretty sure this is currently a dependency for the application to function correctly.

Reply:
Removed.